### ISSUE TITLE
Site install overrides system.site.yml:page.front

### ISSUE THREAD

**Issue by twelvestepdrupal** [twelvestepdrupal/twelvestepintergroup#9]
The [README](https://github.com/anondrupal/anonintergroup/blob/develop/profiles/anonintergroup/README.md) instructions say to install a new site with:

```
$ drush -y si anonintergroup --site-name="Baltimore Central Service Office of Alcoholics Anonymous"
```

While this sets the site name, it also overrides all values in system.site.yml, including page.front, which is pretty important to not override.

**Comment by twelvestepdrupal** [comment]
The problem is that anonintergroup_install() calls standard_install() and standard_install() sets page.front. 

```
  // Set front page to "node".
  \Drupal::configFactory()->getEditable('system.site')->set('page.front', '/node')->save(TRUE);
```

I discussed this at DrupalCon and the solution is to not call standard_install() at all, and just set what we want. standard_install() likely is setting some thing that are already done in configuration and might very well not be needed. So the solution is to copy standard_install() to anonintergroup_install() and remove this one line... but even better, remove as much as we can, because some of the roles and permissions should be settable through our on config/install directory.

**Comment by twelvestepdrupal** [comment]
This is dependent on #7

**Comment by twelvestepdrupal** [comment]
closing, fixed with #7
